.github/workflows/expectfail.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 if $1; then
-    echo "FALSE POSITIVE! Expected failure but succeeded"
-    exit 1
+    echo "Succeeded"
```

Comment:
This file is no more `expectfail.sh`, it's expecting success.
You could simply remove this file and avoid calling it in the yml files. Execute the compiled binaries directly - It will have the same effect.